src/components.js

```diff
@@ -122,6 +125,38 @@ function appComponents() {
     return textArea;
   }
 
+  function newKeyDetails({ id, key, subKey }) {
```

Comment:
Probably not for this release, but I don't agree on building a key details in components. Key details is a specific component of key list. What I believe we should do in components is to create a `newBadge` and probably a new `newLabelWithBadge`. The rest of the keyList specific logic should be built in items and pass down to the footer body. WDYT @JPedroBorges ?

Reply:
Also, just to add to the previous comment, things like key or subkey should decoupled from components. If in the future we add another property related to our data, to be shown on this "header", we should not need to change components.

Reply:
Fixed ✅